assets/rpip-53/merkle-tree-spec.md

```diff
@@ -0,0 +1,164 @@
+4. Concatenate the two hashes (which are both 32 bytes) into a 64-byte array where `left` represents the first 32 bytes and `right` represents the last 32 bytes.
+5. Calculate the `Keccak256` hash of this 64-byte array. **This hash is the value of this branch node.**
+
+Consider the following example:
+
+```
+left = 0xfdbbe597834a953e4c4e50fd8ae8859fd4ae6bf808eb72139ee5a4c224e695f9
+right = 0xad1d32ebc492ff5ad2ab148049de34db5b6d45b9d467823470dffb4c18a4a337
+```
+
+In this case, the concatenated byte array would be `0xad1d32ebc492ff5ad2ab148049de34db5b6d45b9d467823470dffb4c18a4a337fdbbe597834a953e4c4e50fd8ae8859fd4ae6bf808eb72139ee5a4c224e695f9` (note the reordering due to the sorting here).
+
+The `Keccak256` hash of this would be `0xb079b0168e5beba73f17c52b76a614539b242d8efcf6bb99e0dd66a2e251e9e7`, which becomes the value of this branch node.
+
+
+## Leaf Node Values
+
+Start by determining the list of claimers following [the rewards calculation specification](./rewards-calculation-spec.md#determining-claimers-and-claimer-rewards).
+
+Each leaf node of the tree will be the the `Keccak256` hash of a single claimer's metadata.
+The structure for this metadata is as follows:
+
+`metadata = address[20] ++ network[32] ++ totalRPL[32] ++ totalETH[32]`
+
+where `address[20]` refers to a byte array of size 20 bytes, and the `++` operator refers to byte array concatenation.
+
+Each of these values is defined as follows:
+
+- `address` is the standard 20-byte Ethereum address of the claimer.
+- `network` is a 256-bit (32-byte) unsigned integer representing which Rocket Pool network the rewards will be available on.
+  - `0` represents Mainnet.
+  - More values will be added as we integrate support for various L2s to the rewards system. Currently, `0` is the only legal value for this.
+- `totalRPL` is a 256-bit (32-byte) unsigned integer representing how much RPL the claimer is eligible to claim for this rewards period.
+  - This value is the sum of the RPL rewards provided for staking minipools and the RPL rewards provided for ODAO membership.
+  - The RPL value is expected in **wei**, so rewards of `2` RPL would be represented as `2000000000000000000` (in decimal).
+- `totalETH` is a 256-bit (32-byte) unsigned integer representing how much ETH is eligible to claim from the Smoothing Pool for this rewards period.
+  - The ETH value is expected in **wei**, so rewards of `2` ETH would be represented as `2000000000000000000` (in decimal).
+
+In total, the metadata will be an array of **116 bytes**.
+
+For example, say there were a claimer with the following values in its metadata:
+- Address: `0x8B0EF9f1932A2e44c3D27bE4C70C3BC07A6A27B3`
+- Network: `0`
+- Total RPL: 1.5 RPL (`1500000000000000000` in wei which is `14d1120d7b160000` in hexadecimal)
+- Total ETH: 2.5 ETH (`2500000000000000000` in wei which is `22b1c8c1227a0000` in hexadecimal)
+
+This tree node would have the following metadata byte array (in hexadecimal):
```

Comment:
Or maybe because you're trying to talk about the content of the tree node itself... you should still maybe say "leaf node"? internal tree nodes don't have this stuff..